### ISSUE TITLE
Fill indicator displayed in Windows Chrome without touch screen

### ISSUE THREAD

**Issue by johannesh2** [vaadin/spreadsheet#430]
I don't have a touch device. I open https://demo.vaadin.com/spreadsheet and get the fill indicator button.

**Comment by OlliTietavainenVaadin** [comment]
Happens to me as well. Only Win10 and only in Chrome, not IE, Edge or FireFox. Does not happen in Win7 + Chrome. Reproducible on bare minimum project with nothing but a spreadsheet.

**Comment by alvarezguille** [comment]
Seems related to issues [467934](https://bugs.chromium.org/p/chromium/issues/detail?id=467934) and [676808](https://bugs.chromium.org/p/chromium/issues/detail?id=676808) in Chromium project
@OlliTietavainenVaadin could you test with latest Chrome Canary on same windows? Thanks!

**Comment by OlliTietavainenVaadin** [comment]
Yes, just updated Canary and it's still happening. Here's a video: https://drive.google.com/open?id=0B6t39Qrx19miWDBUR3k4SlFlUUk

**Comment by OlliTietavainenVaadin** [comment]
Issue still happens on Win 10 + Chrome. The issue also occurs on Windows 10 + Firefox 53.0.2 (32-bit). It also makes it impossible to resize columns.

**Comment by OlliTietavainenVaadin** [comment]
Windows 10 + IE11 and Edge are unaffected, though.
